EXAMPLES.md

```diff
@@ -0,0 +1,98 @@
+# Very common words; (in lists: All, Default, Common, Very Common)
+
+```
+attempt base borrows clever cold concerned contribution decide deletes
+easiest inclined mine natural obviously opportunity organized pain
+potential signed significance standing survey this training trick
+```
+
+To get _only_ words like these, import this file:
+
+```js
+import veryCommonWords from 'wordlist-english/veryCommon';
+```
+
+Words like these will be included in each of these files:
+
+```
```

Comment:
@jacksonrayhamilton This code block should have the language defined as `js` so you get proper syntax highlighting when viewing the file in GitHub.

```suggestion
```js
```

Reply:
Thanks for the catch!  Updated in all places.